sql/moz-fx-data-shared-prod/addons_derived/dataset_metadata.yaml

```
@@ -2,10 +2,15 @@ friendly_name: Addons Derived
 description: |-
   Derived data used for Addons related analysis.
 dataset_base_acl: derived
-user_facing: true
+user_facing: false
 labels: {}
+default_table_workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:mozilla-confidential
```

I'd think you'd want the default table access to be the same as the dataset access.
```suggestion
  - workgroup:mozilla-confidential
  - workgroup:amo/prod
```